hook_test.go

```diff
@@ -83,13 +117,18 @@ var _ = Describe("Hook", func() {
 				err := hook.Subscribe(context.Background(), "http://localhost", "[")
 				Expect(err).To(Succeed())
 				err = hook.DoSend(context.Background(), "message", "test")
-				Expect(err).ToNot(Succeed())
-				Expect(err.Error()).To(Equal("send failed"))
+				Expect(err).To(Succeed())
```

Comment:
Why this test succeed ?
It shouldn’t as the scope isn’t a good regex

Reply:
We need to raise an error in case isInScope() condition is not respected
That's not the case here

Reply:
Ok that means here is succeeded but not sent ?

Reply:
yep, we can raise an error here https://github.com/w6d-io/hook/blob/ae14f0b4c93864d768011d3536d970c085bcf6e7/hook.go#L65